spring-cloud-dataflow-shell-core/src/test/java/org/springframework/cloud/dataflow/shell/command/CounterCommandsTests.java

```diff
@@ -37,42 +37,43 @@
  */
 public class CounterCommandsTests extends AbstractShellIntegrationTest {
 
-	private static MetricRepository repository;
+	// TODO: BOOT2 disabled metric stuff
+	//	private static MetricRepository repository;
 
 	@BeforeClass
 	public static void setUpOnce() {
-		repository = applicationContext.getBean(MetricRepository.class);
+		//		repository = applicationContext.getBean(MetricRepository.class);
 	}
 
 	@Before
 	@After
 	public void cleanSlate() {
-		repository.findAll().forEach(m -> repository.reset(m.getName()));
+		//		repository.findAll().forEach(m -> repository.reset(m.getName()));
 	}
 
-	@Test
-	public void testCounterInteractions() {
-		Table table = metrics().listCounters();
-		assertThat(table.getModel().getColumnCount(), is(1));
-		assertThat(table.getModel().getRowCount(), is(1));
-
-		repository.set(new Metric<>("counter.foo", 12.0d));
-		repository.set(new Metric<>("counter.bar", 42.0d));
-		table = metrics().listCounters();
-		// Test alphabetical order
-		assertThat(table.getModel().getColumnCount(), is(1));
-		assertThat(table.getModel().getValue(1, 0), is("bar"));
-		assertThat(table.getModel().getValue(2, 0), is("foo"));
-
-		String value = metrics().displayCounter("foo");
-		assertThat(value, is("12"));
-
-		String message = metrics().resetCounter("foo");
-		assertThat(message, is("Deleted counter 'foo'"));
-
-		table = metrics().listCounters();
-		assertThat(table.getModel().getColumnCount(), is(1));
-		assertThat(table.getModel().getRowCount(), is(2));
-	}
+	//	@Test
```

Comment:
create issue.  Is this related to shell issues?

Reply:
I think this was related to metric issues and didn't invest time as sabby wasn't sure if analytics are going to kept in place. I'll create follow up story.